renovate.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ],
+  "ignorePaths": [
+    "**/ui/**"
+  ],
+  "regexManagers": [
+    {
+      "fileMatch": [
+        "build-images.sh"
+      ],
+      "matchStrings": [
+        "NC_VERSION=(?<currentValue>[^\\s]+)"
+      ],
+      "depNameTemplate": "nextcloud",
+      "datasourceTemplate": "docker"
+    },
+    {
+      "fileMatch": [
+        "build-images.sh"
+      ],
+      "matchStrings": [
+        "docker\\.io\/node:(?<currentValue>[^\\s]+)"
+      ],
+      "depNameTemplate": "node",
+      "datasourceTemplate": "docker"
+    },
+    {
+      "fileMatch": [
+        "build-images.sh"
+      ],
+      "matchStrings": [
+        "docker\\.io\/redis:(?<currentValue>[^\\s]+)"
+      ],
+      "depNameTemplate": "redis",
+      "datasourceTemplate": "docker"
+    },
+    {
+      "fileMatch": [
+        "build-images.sh"
+      ],
+      "matchStrings": [
+        "docker\\.io\/mariadb:(?<currentValue>[^\\s]+)"
+      ],
+      "depNameTemplate": "mariadb",
+      "datasourceTemplate": "docker"
+    },
+    {
+      "fileMatch": [
+        "build-images.sh"
+      ],
+      "matchStrings": [
+        "docker\\.io\/nginx:(?<currentValue>[^\\s]+)"
+      ],
+      "depNameTemplate": "nginx",
+      "datasourceTemplate": "docker"
+    },
+    {
+      "fileMatch": [
+        "test-module.sh"
+      ],
+      "matchStrings": [
+        "ghcr\\.io\/marketsquare\/robotframework-browser\/rfbrowser-stable:(?<currentValue>[^\\s]+)"
+      ],
+      "depNameTemplate": "ghcr.io/marketsquare/robotframework-browser/rfbrowser-stable",
+      "datasourceTemplate": "docker"
```

Comment:
Clean up robotframework matcher

```suggestion
```

Reply:
Let's leave it, I'm just gonna lock the dependency to 10.1.z, so if any bugfix/security fix is released, we'll be notified.